x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -178,7 +182,39 @@ export const OnboardingFlowForm: FunctionComponent = () => {
             <EuiCheckableCard
               id={`${radioGroupId}_${option.id}`}
               name={radioGroupId}
-              label={<EuiText css={{ fontWeight: 'bold' }}>{option.label}</EuiText>}
+              label={
```

Comment:
We probably need to rase this with EUI team that there is no way to specify a short accessibility label for a `CheckableCard`, because now it will be labeled by the whole `label` content.

cc @elastic/eui-team
For context, previously we moved the content of `CheckableCard` out of `label` into `children` to have a concise accessibility label for the card, but then user cannot click on the card's content to select the card anymore. Maybe there is a workaround for this already to have the full card respond to click and at the same time to have a short accessibility label?

Reply:
The easiest way to accomplish this is to add a `<span aria-hidden>` wrapper around content that you do not want to have read out by screen readers. However, I would strongly caution thinking twice before doing this, as you now have information that screen reader users literally cannot access which is available to sighted users but not SR users.

I understand the desire for a concise/shorter SR label, but this is not always something we should do or decide on behalf of SR users. Typically we want information to be 1:1 for SR and non SR users, and SR users can easily pause or skip narration if they decide that the extra information isn't useful to them.

@alexwizp do you mind chiming in here to confirm/add recommendations?

Reply:
Thanks @cee-chen and good point regarding maintaining information. Let's keep the description visible to screen readers but hide the icons. They are mainly there to strike a balance between text and visual elements on the page but don't add value to screen readers. The descriptions on the other hand add more context and explanation so it's worth keeping that in for screen reader users.